contracts/script/GenerateAlloc.s.sol

```diff
@@ -239,8 +242,27 @@ contract GenerateAlloc is Script {
         // Allocation
         if (block.chainid == MAINNET_CHAIN_ID) {
             // TBD
+        } else if (block.chainid == 1315) {
+            // Odyssey devnet alloc
+            vm.deal(0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266, 100000000 ether);
+            vm.deal(0xf398C12A45Bc409b6C652E25bb0a3e702492A4ab, 100000000 ether);
+            vm.deal(0xEcB1D051475A7e330b1DD6683cdC7823Bbcf8Dcf, 100000000 ether);
+            vm.deal(0x5518D1BD054782792D2783509FbE30fa9D888888, 100000000 ether);
+            vm.deal(0xbd39FAe873F301b53e14d365383118cD4a222222, 100000000 ether);
+            vm.deal(0x00FCeC044cD73e8eC6Ad771556859b00C9011111, 100000000 ether);
+            vm.deal(0xb5350B7CaE94C2bF6B2b56Ef6A06cC1153900000, 100000000 ether);
+            vm.deal(0x13919a0d8603c35DAC923f92D7E4e1D55e993898, 100000000 ether);
+        } else if (block.chainid == 1516) {
+            // Odyssey testnet alloc
+            vm.deal(0x5687400189B13551137e330F7ae081142EdfD866, 200000000 ether);
+            vm.deal(0x56A26642ad963D3542DdAe4d8fdECC396153c2f6, 200000000 ether);
+            vm.deal(0x12cBb8F6F2F7d48bB22B6A1b12452381A45bEb7c, 100000000 ether);
+            vm.deal(0xD26078bA39afccec71E0D68a151a853d21950FF0, 200000000 ether);
+            vm.deal(0xcA93A8f7a3971D208670876202D8353Ca3D6869a, 200000000 ether);
+            vm.deal(0x8Ffc89da28DD2F5f7582B0459505E9a615623791, 10000000 ether);
+            vm.deal(0xE8DA8e345Ab1556E5DeE19F9c369C827561Ff712, 10000000 ether);
         } else {
-            // Testnet alloc
+            // test alloc
```

Comment:
Would it be better to keep the previous comment so all of them are following the `Abcnet alloc` format?

Reply:
sounds good